src/styles/pages/style.scss

```diff
@@ -0,0 +1,844 @@
+body {
+  font-size: 16px;
+  line-height: 1.5;
+  color: #F5F8FB;
+  font-family: 'Bakbak One', Arial, sans-serif;
+  background: #0D0D15;
+}
+
+/* Bakbak One for headings */
+.bakbak-one-regular {
+  font-weight: 400;
+  font-style: normal;
+}
+
+/* Poppins for body text */
+.poppins-medium {
+  font-family: 'Poppins', Arial, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+/* Header */
+.header {
+  width: 1728px;
```

Comment:
Remove fixed widths and heights: Avoid hardcoding width and height values for layout blocks unless necessary. This limits flexibility and responsiveness.